index.html

```diff
@@ -99,77 +138,90 @@ <h3 class="projects-title">Project names goes here</h3>
                               and have a project you need coded,
                               don’t hesitate to contact me.</p>
-      <button type="button" class="button-box">Get my CV
-           <img id="download-image" src="Images SVG/ic_download.svg"></button>  
-    
-      <div id="language-template">
-        <p class="templates-title">Languages</p>
-        <ul class="list-container">
-          <li class="template-item">JavaScript(ES5/ES6)</li>
-          <li class="template-item">Ruby</li>
-          <li class="template-item">HTML</li>
-          <li class="template-item">CSS</li>
-        </ul>
-      </div>
+        <button type="button" class="button-box">Get my CV
+             <img id="download-image" src="Images SVG/ic_download.svg"></button>  
+      </div> 
+      
+      <div id="templates-container">
+        <div id="language-template">
+          <p class="templates-title">Languages</p>
+          <ul class="list-container">
+            <li class="template-item">JavaScript(ES5/ES6)</li>
+            <li class="template-item">Ruby</li>
+            <li class="template-item">HTML</li>
+            <li class="template-item">CSS</li>
+          </ul>
+        </div>
         
-      <div id="frameworks-template">
-        <p class="templates-title">Frameworks</p>
-        <ul class="list-container">
-          <li class="template-item">React.js</li>
-          <li class="template-item">Ruby on rails</li>
-          <li class="template-item">RSpec</li>
-          <li class="template-item">Capibara</li>
-        </ul>
-      </div>
+        <div id="frameworks-template">
+          <p class="templates-title">Frameworks</p>
+          <ul class="list-container">
+            <li class="template-item">React.js</li>
+            <li class="template-item">Ruby on rails</li>
+            <li class="template-item">RSpec</li>
+            <li class="template-item">Capibara</li>
+          </ul>
+        </div>
         
-      <div id="skills-template">
-        <p class="templates-title">Skills</p>
-        <ul class="list-container">
-          <li class="template-item">Database Management</li>
-          <li class="template-item">CLI</li>
-          <li class="template-item">API Design</li>
-          <li class="template-item">Version Control</li>
-          <li class="template-item">Web Development</li>
-        </ul>
-      </div>
+        <div id="skills-template">
+          <p class="templates-title">Skills</p>
+          <ul class="list-container">
+            <li class="template-item">Database Management</li>
+            <li class="template-item">CLI</li>
+            <li class="template-item">API Design</li>
+            <li class="template-item">Version Control</li>
+            <li class="template-item">Web Development</li>
+          </ul>
+        </div>
+        <div id="dot-grid-template"><img src="Images SVG/image_dots.svg" alt=""></div>
+      </div>     
     </section>
```

Comment:
The about myself section needs to be centered with evenly distributed space/margin to the left and right side. However, your design seems to be shifted to the left side. Kindly check and ensure that it follows the margin, paddings, and other dimensions of the actual design in order to achieve a near perfect project that looks like the actual design :1st_place_medal: 


### Your design 

![Screenshot from 2023-04-08 00-55-33](https://user-images.githubusercontent.com/65233860/230696968-cde1cc8f-c447-4b52-a331-4b1ddcb73a93.png)



### Actual design 

![Screenshot from 2023-04-08 01-14-20](https://user-images.githubusercontent.com/65233860/230696970-b45c56c9-d8c3-49b3-b755-9855aaf403a3.png)